src/ac_copilot_trainer/ac_copilot_trainer.lua

```
@@ -639,8 +661,34 @@ local function splineForwardDelta(carSpline, ptSpline)
   return d
 end
 
+local function ensureCornerIniLoaded(sim0)
```

Luacheck (1.2.0)

[warning] unused argument 'sim0'

(W212)